### PR TITLE
chore(web): Update keymanweb-osk.ttf to v. 3.0

### PR DESCRIPTION
**Pull request by darcywong00**
From WS Tech's release notes of [SymChar v3.000](https://github.com/silnrsi/font-symchar/releases/tag/v3.000)
> This is the first release of a new set of fonts that can be used by applications to show symbols that represent 'invisible' characters (such as NBSP). There are also symbols for many common keyboard keys and a few social media sources.
>
>There are two variants, released in separate packages:
>
> * SymChar - The package for general use, with all symbols in the PUA
> * SymCharK - A version that adds some basic Latin letters and symbols, specifically for use by Keyman. This incorporates all the symbols from the KeymanWeb-OSK font.

On Slack, @jvgaultney also adds:
> The only difference from KeymanWeb-OSK is that the default line metrics are a tiny bit larger (by 1%) - although the glyphs themselves are unchanged in size. This could impact CSS alignment, but only by a trivial amount.

---------

This PR takes SymCharK and updates the keymanweb-osk.ttf font used in KeyamnWeb

User Testing updated to use a custom test9469 keyboard which uses special characters on the touch keyboard keycaps and longpresses.

## User Testing
Note: the test9469 keyboard only displays the special characters on the keycaps. None of the keys have output defined.

* **TEST_WEB** - Verifies special characters render on the OSK keycaps and longpress keys
Use the "Test special characters rendering with keymanweb-osk.ttf (#9469)" test page link on a mobile form-factor device in landscape orientation.
1. Load the test page and display the OSK. The two backspace keys that look identical are `*LTRBkSp*` and the regular `*BkSp*`.
Note: `*Tab*` and `*TabLeft*` appear different sizes, but I haven't investigated to see if the previous font did that too.
3. Verify the keycaps on the default layer display
4. Longpress on the <kbd>Shift</kbd> button to display longpress options for other "layers".
5. Verify the shift longpresses render fine (Should not see `*` characters)
6. Longpress on the <kbd>Return</kbd> button to display longpress options for Return.
7. Verify the return longpresses render fine. (The one that looks like normal return is actually `*LTREnter*`)

kmw-osk shift layers
Long-presses on shift layer

* **TEST_ANDROID** - Verifies special characters render on the OSK keycaps and longpress keys
Setup - Install the PR build of KeyboardHarness on an Android 5.0 Lollipop device/emulator
1. Launch the KeyboardHarness app
2. Use the Globe key to select "test9469" keyboard. (There's several test keyboards to cycle through
3. Verify the keycaps on the default layer display
4. Press the <kbd>Shift</kbd> button to display switch to the shift layer. 
5. On the shift layer, verify all the special characters on the keycaps render fine
(The one that looks like normal return is actually `*LTREnter*`)

android shift keycaps
KeyboardHarness with test9469 keyboard on the shift layer 

* **TEST_IOS** - Verifies special characters render on the OSK keycaps and longpress keys
Setup - Install the PR build of Keyman for iPhone and iPad
1. Launch Keyman app
3. Verify the OSK is fine

### CODE REVIEW

**Review by rc-swag — APPROVED** [review]
RSLGTM

**Comment by mcdurdin** [comment]
Important to verify that the key caps do look correct with the new font, as it is possible that the 1% change in size could bump a margin or something

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_WEB (PASSED):** Tested with the test page link and here is my observation: 1. Verified that the KMW keyboard is active and it is possible to type using the OSK. 

   


* **TEST_ANDROID (PASSED):** Installed it in Android 5.1 Lollipop emulator and here is my observation : 1. Able to open the Keyman In-App. 2.Verified that the OSK is working fine. 
   
  


* **TEST_IOS (PASSED):** Installed the PR build in iPhone and iPad Simulator and here is my observation: 1. Able to open the Keyman In-App. 2. Verified that the OSK is working fine.

**Comment by mcdurdin** [comment]
Just noting that I think we need to verify the symbols that are in the font are working correctly, as I am not sure the basic test we've done here covers much of that.

**Comment by darcywong00** [comment]
> Just noting that I think we need to verify the symbols that are in the font are working correctly, as I am not sure the basic test we've done here covers much of that.

I've created test9469 keyboard to render all the special characters, and updated the user testing steps to verify them.
TODO
- [ ] How to test iOS with the keyboard?

The longpress keys on Android also don't render. Is that Java  or KMW code?

**Comment by mcdurdin** [comment]
> Is that Java or KMW code in #9187?

A separate issue needs to be tracked for this:

1. transforming the `*token*` patterns in the longpress captions in Java code (can't be done KMW side, because legit keyboards could use the same codepoints)
2. using the kmw font where the `*token*` captions are found (and only there)

Resolution for this depends on our decision on Android Longpress in the upcoming Keyman Planning Conference.

**Comment by bharanidharanj** [comment]
* **TEST_ANDROID (FAILED):** Tested with the attached PR build (Keyboard Harness.apk) in Android 5.1 / API 22 emulator and here is my observation: 1. Got an Keyboard error message after opening the Keyboard Harness app. 2. Did not see the mentioned 'test 9469' keyboard in the Keyboard picker menu. Seems to be an issue. 
..Keyboard error message

   


..test 9469 keyboard is missing

**Comment by darcywong00** [comment]
Had to push up changes for updated test keyboard
@keymanapp-test-bot retest

File: web/src/test/manual/web/index.html [hunk 1]
```diff
@@ -64,6 +64,7 @@ <h2><a href="./caps-lock-layer-3620/">Test Caps Lock Layer (#3620)</a></h2>
   <h2><a href="./start-of-sentence-3621/">Test Start of Sentence (#3621)</a></h2>
   <h2><a href="./start-of-sentence-3621/">Test start of sentence keyboard rules (#5963)</a></h2>
   <h2><a href="./issue6005/">Tests predictive text & other handling of rule matching when the final rule group does not match (#6005)</a></h2>
+  <h2><a href="./issue9469/">Test special characters rendering with keymanweb-osk.ttf (#9469)</a></h2>
```
Comment (darcywong00) [inline web/src/test/manual/web/index.html:67]: @mcdurdin - if I add a test page here, do I need to update something else so @bharanidharanj is able to get the [test artifact](https://build.palaso.org/repository/download/Keymanweb_TestPullRequests/403532:id/src/test/manual/web/)?

Comment (darcywong00) [reply web/src/test/manual/web/index.html:67]: I just pushed the build/test9469.js file. We'll see if that did it...

Comment (darcywong00) [reply web/src/test/manual/web/index.html:67]: Yep, the test page shows now

**Comment by bharanidharanj** [comment]
* **TEST_ANDROID (PASSED):** Retested with the updated PR build (Keyboard Harness_test9469) in Android 5.1 / API 22 emulator and here is my observation: 1. Got an keyboard error message after opening the keyman app. (PFA). 2. Able to see the keycaps appear correctly in the default layer. 3. Verified all the special characters on the keycaps render fine, on the shift layer. 


..Keyboard error while opening keyman app

   


..default layer and shift layer appear as expected

**Comment by bharanidharanj** [comment]
* **TEST_WEB (FAILED):** Tested with the given page link on a mobile form-factor device in landscape orientation and here is my observation: 1. Did not see any key names on the keys. Not able to do further testing with this view.

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_IOS (PASSED):** Tested with the attached PR build (Keyman 17.0.161-alpha-test-9469)  in iPhone and iPad Simulators and here is my observation : 1. Verified that the OSK is working as expected in both shift layer and longpress. Seems to be working fine. 

..OSK is working fine

**Comment by darcywong00** [comment]
> * test page link on a mobile form-factor device in landscape orientation

You tested in desktop-form factor which doesn't have the touch keyboard defined.
The setup is
> test page link on a **mobile form-factor** device in landscape orientation

which is accessed with F12 in Chrome

@keymanapp-test-bot retest TEST_WEB

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_WEB (PASSED):** Retested this PR as per Darcy's suggestion and here is my observation: 1. Verified that the two backspace keys are appear as expected. Also, noticed that the *Tab* and *TabLeft* keys are appear in different sizes. 2. Verified that the keycaps on the default layer appear correctly. 3. Longpressing the shift button shows the longpress options for other "layers". However, the keypad seems to be in white background and I did not see the key button frame in this layer. 4. Verified that the shift longpresses render fine. (ie., It did not show any * characters in this layer). 5. Long pressing the Return button displays the long press options for Return. Verified that the return longpresses render fine. 


..two Backspace keys and two Tabs

   


..Longpress options appear in the Shift layer (appear with white background)

   


..two Return buttons

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.163-alpha](https://keyman.com/downloads/releases/alpha/17.0.163)

**Pull request merged**